src/components/control-panel-pages/employees/employees-list/employees-list.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./employees-list.css";
+import endpointUrl from "../../../../config";
+
+const handleDelete = _id => {
+  fetch(`${endpointUrl}/delete-user`, {
+    method: "POST",
+    body: JSON.stringify({ _id }),
+    headers: {
+      "Content-Type": "application/json"
+    }
+  }).then(res =>
+    res.status === 200 ? (window.location = "/employees") : null
```

Comment:
Why are you using `window.location` here instead of `history.push()`? It would be better to have consistency through the application

Reply:
history.push() not rendering the component (it's not reload the page)